index.html

```diff
@@ -8,9 +8,9 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <div class="container" style="background-image:url('/home/ignatius/Documents/Microverse Projects/Portfolio/images/background-image.png');">
+    <div class="container">
         <div class="toolbar">
-            <p class="heading-1" style="color:blue ;">My logo</p>
+            <p class="heading">My logo</p>
         </div>
```

Comment:
- [ ] [REQUIRED] Good job so far! The `logo` (if the design has one) should be on the left and has an `a` link (`a` tag). This is for three reasons: `convention`, `semantics`, and `user` experience(`UX`). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info and the `hamburger` button should be on the `right` and has a `nav` tag. The `nav` tag is for the `semantics` reason and the right position for convention and user experience for mobile devices. Please implement my suggestions and also previous reviewers 🙏🏼 


```HTML
<header>
    <a href...>Website Logo</a>
    <nav>
        <!-- Main Navigation inside <header> -->
        <ul>
            <li></li>
        </ul>
    </nav>
</header>
```